src/components/forms/SearchInput.vue

```
@@ -0,0 +1,38 @@
+  set: (value: string) => emit('update:modelValue', value),
+})
+</script>
+
+<template>
+  <div class="relative w-full">
+    <input
+      v-model="localValue"
+      type="text"
+      :placeholder="placeholder"
+      :disabled="disabled"
+      :class="['w-full pl-10 d-input', props.class]"
```

ESLint

[error] Static class "w-full pl-10 d-input" should be in a static `class` attribute.

(vue/prefer-separate-static-class)